README.adoc

```diff
@@ -127,12 +140,15 @@ followed by fields that correspond to metadata fields.
 *The `description` field is optional and describes the purpose of the metric
```

Comment:
There should be a blank line before this list for the list to properly show up

Reply:
Done

Reply:
The blank line is fine.
But let's move lines 87-88 up before the png....maybe like this...

To see only the application metrics, point your browser to the `https://localhost:9443/metrics/application` URL. Because these metrics are application-related, you need to point first to the `http://localhost:9080/inventory/systems` URL, or the metrics do not appear.

I think I misunderstood the last sentence and how it fits in context the last time I read it.

Reply:
Done